sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 
 - Remove `RecognizedReceipts` Class.
 - `begin_recognize_receipts` and `begin_recognize_receipts_from_url` now return `RecognizedForm`.
+- `requested_on` renamed to `training_started_on` and `completed_on` renamed to `training_completed_on` on `CustomFormModel`
```

Comment:
should we hold off on this change until we discuss about the possibility of renaming `CustomFormModel` to something training related?

Reply:
seems we decided in the meeting today - `CustomFormModel` it is!